pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -642,11 +642,23 @@ protected FunctionDetails convert(FunctionConfig functionConfig)
 
         @Parameter(names = "--brokerServiceUrl", description = "The URL for the Pulsar broker")
         protected String brokerServiceUrl;
+        
+        @Parameter(names = "--clientAuthPlugin", description = "Client authentication plugin using which function-process can connect to broker")
+        protected String clientAuthPlugin;
+        
+        @Parameter(names = "--clientAuthParams", description = "Client authentication param")
+        protected String clientAuthParams;
+        
+        @Parameter(names = "--use_tls", description = "Use tls connection\n")
+        protected boolean useTls;
+
+        @Parameter(names = "--tls_allow_insecure", description = "Allow insecure tls connection\n")
+        protected boolean tlsAllowInsecureConnection;
```

Comment:
Do we not need to add `tlsTrustCertsFilePath` and `tlsHostnameVerificationEnable` options?

Reply:
we don't need `tlsTrustCertsFilePath` because these configs will be used to create `Pulsar-Consumer` only for which we already have plugin and params. I am not sure `tlsHostnameVerificationEnable` can be useful option other than proxy but still I will add it.

Reply:
added it.